package.json

```diff
@@ -21,8 +21,12 @@
     "atom": ">=0.174.0"
   },
   "dependencies": {
-    "atom-linter": "^3.0.0"
+    "atom-linter": "^3.0.0",
+    "atom-package-deps": "^3.0.0"
```

Comment:
Latest is currently [v3.0.6](https://github.com/steelbrain/package-deps/releases/tag/v3.0.6).